### PR TITLE
Adding CI to Shaded Hills.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Really need to enable the branch tests for this one.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Seems like when I pushed the base branch I included some commits I shouldn't have, causing some conflicts/clobbering.

**Comment by out-of-phaze** [comment]
> Really need to enable the branch tests for this one.

I don't think we can, since it's not on all branches? I'm not sure. Might have to make a special test action that only runs on this branch...

**Comment by MistakeNot4892** [comment]
CI runs off the workflow in the branch itself, so should be fine. Seems to be working on this one anyway.

File: maps/shaded_hills/jobs/_jobs.dm [hunk 1]
```diff
@@ -1,30 +1,37 @@
-/decl/hierarchy/outfit/shaded_hills
-	abstract_type = /decl/hierarchy/outfit/shaded_hills
+/datum/map/shaded_hills
+	id_hud_icons = 'maps/shaded_hills/icons/hud.dmi'
+
+/decl/hierarchy/outfit/job/shaded_hills
+	name = "Shaded Hills Outfit"
+	abstract_type = /decl/hierarchy/outfit/job/shaded_hills
```
Comment (out-of-phaze) [inline maps/shaded_hills/jobs/_jobs.dm:6]:
```suggestion
	abstract_type = /decl/hierarchy/outfit/job/shaded_hills
	id_type = null
	pda_type = null
	l_ear = null
```

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

File: nebula.dme [hunk 2]
```diff
@@ -3815,8 +3820,8 @@
 #include "maps\random_ruins\exoplanet_ruins\spider_nest\spider_nest.dm"
 #include "maps\random_ruins\exoplanet_ruins\tar_anomaly\tar_anomaly.dm"
 #include "maps\random_ruins\space_ruins\space_ruins.dm"
-#include "maps\shaded_hills\shaded_hills.dm"
 #include "maps\shaded_hills\shaded_hills_define.dm"
+#include "maps\tradeship\tradeship.dm"
```
Comment (out-of-phaze) [inline nebula.dme]:
```suggestion
#include "maps\shaded_hills\shaded_hills.dm"
#include "maps\shaded_hills\shaded_hills_define.dm"
```

**Review by out-of-phaze — CHANGES_REQUESTED** [review]

**Review by out-of-phaze — APPROVED** [review]

**Pull request merged**